### PR TITLE
feat: accordion atom

### PR DESCRIPTION
**Pull request by 0c370t**

### CODE REVIEW

**Comment by 0c370t** [comment]
Task linked: [CU-2q4p53y Accordion Atom](https://app.clickup.com/t/2q4p53y)

**Review by mrdonovannn — APPROVED** [review]
Looks good from my end.

**Review by CodeRedJack — APPROVED** [review]

File: clients/web/src/lib/accordion/Accordion.stories.svelte [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+</Template>
+
+<Story name="Default" args={{ open: false, flush: false }} />
+<Story name="Flush" args={{ open: false, flush: true }} />
+
+<style lang="postcss">
+    div {
+        @apply bg-gray-900 p-12;
+    }
+</style>
```
Comment (zachstence) [inline clients/web/src/lib/accordion/Accordion.stories.svelte:37]: Default background color can be configured in storybook settings

https://storybook.js.org/docs/react/essentials/backgrounds

**Review by zachstence — APPROVED** [review]

File: clients/web/src/lib/accordion/Accordion.stories.svelte (hunk 1 above)
Comment (zachstence) [reply clients/web/src/lib/accordion/Accordion.stories.svelte:37]: Nvm this ain't storybook lol

**Pull request merged**